fe/src/main/cup/sql_parser.cup

```diff
@@ -967,6 +967,10 @@ create_stmt ::=
     {:
         RESULT = new AlterTableStmt(tableName, Lists.newArrayList(new CreateIndexClause(tableName, new IndexDef(indexName, cols, indexType, comment), false)));
     :}
+    | KW_CREATE KW_INDEX ident:indexName opt_index_type:indexType KW_ON table_name:tableName LPAREN ident_list:cols RPAREN opt_comment:comment
```

Comment:
I think we can not support this syntax.
It is enough to give user one option, no need to give users multiple options.